packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Stats.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Stack, Text } from '@codesandbox/components';
+import { HeartIcon, ViewIcon, ForkIcon } from './icons';
+
+export const Stats = ({ sandbox }) => (
```

Comment:
Should we make a `Stats` component in package? 😅

Reply:
I thought about that, but should we? it's all reusable components.

I can make it if you think it makes sense :)

Reply:
I think it makes sense because we'll need it in profiles and embed too!

Reply:
Moved :)